docs/source/private/prep_and_debug.rst

```diff
@@ -0,0 +1,51 @@
+.. _prep_and_debug:
+
+***********************************************
+FAQ: Preparing and Debugging OpenFE Simulations
+***********************************************
+
+This FAQ contains a list of issues you may encounter when preparing and
+running simulations with OpenFE. This list will be updated as feedback
+is provided by benchmark partners.
+
+
+1. Ligand Alignment
+*******************
+
+Ligands in a series should be pre-aligned when pass to OpenFE tools.
+This means that regions of the ligand that should be part of a common
+core, should have the same coordinates.
+
+In these benchmarks we will use the Kartograf atom mapper. Any ligand
+regions which do not align in 3D space will be unmapped and treated
+as unique atoms in the alchemical topology.
+
+
+2. Protein Structures Ready for Simulation
+******************************************
+
+OpenFE does not run a very long equilibration during the RBFE calculation.
+As such, a protein structure with clashes will likely fail with the Protocol
+reporting "NaN" errors.
+
+You should make sure that your protein structure is ready for MD simulation
+before passing it to OpenFE.
+
+
+3. All My Complex Edges Fail But Not The Ligands
+************************************************
+
+This indicates a likely issue with the protein structure, either a clash
+with the ligand, or a clash within the protein structure itself.
+
+To check this we suggest running a conventional MD simulation and seeing if the system remains stable.
+You can use the `OpenFE MD Protocol for this purpose <https://docs.openfree.energy/en/stable/tutorials/md_tutorial.html>`.
+
+
+4. My Ligand Transformations Are Failing
+****************************************
+
+This likely indicates a bad mapping.
+
+We would encourage you to `visualize your mappings <https://docs.openfree.energy/en/stable/cookbook/ligandnetwork_vis.html>`.
```

Comment:
```suggestion
We would encourage you to `visualize your mappings <https://docs.openfree.energy/en/stable/cookbook/ligandnetwork_vis.html>`_.
```

Reply:
This cookbook used as example an MST and LomapAtomMapper, it might be nice to have an example on how to load in the `ligand_network.graphml` and look at the mapping from there, but may also not be required.

Reply:
I unfortunately do not have the time to make one, is this something you might have time for?

Reply:
I just checked, I have an example notebook for loading in the transformation .json file and displaying a mapping (the one we sent to Christina a while ago). Would that be helpful?